### PR TITLE
feat(console): add standard connector

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

New feature for AC: add standard connector, with a new form.

## Testing

Local tested: manually change `GitHub`'s `isStandard` to true:

image
image

### CODE REVIEW

**Review by simeng-li — COMMENTED** [review]
overall LGTM, leaves the final stamp to @darcyYe  as he has more context

File: packages/console/src/types/guide.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 export type GuideForm = {
   redirectUris: string[];
   postLogoutRedirectUris: string[];
-  connectorConfigJson: string;
```
Comment (xiaoyijun) [inline packages/console/src/types/guide.ts:4]: It seems that we should not delete this prop; for it's used in the connector details page.

File: packages/console/src/pages/Connectors/components/Guide/Form.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  return (
+    <div>
+      {isStandard && (
+        <>
+          <FormField isRequired title="connectors.guide.name">
+            <TextInput
+              placeholder={t('connectors.guide.name')}
+              {...register('name', { required: true })}
+            />
+            <div className={styles.tip}>{t('connectors.guide.name_tip')}</div>
```
Comment (xiaoyijun) [inline packages/console/src/pages/Connectors/components/Guide/Form.tsx:37]: This looks like a tooltip; maybe sync this information with the designer.

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/Guide/Form.tsx:37]: Per discussion, a design system project will cover this.

File: packages/console/src/types/guide.ts (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/types/guide.ts:4]: No, the connector details page doesn't use this type.

**Review by darcyYe — APPROVED** [review]

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Pull request merged**